.github/workflows/ci-jdk11.yml

```
@@ -22,8 +22,24 @@ name: Java CI with Maven On JDK 11
 on:
   push:
     branches: [ "master" ]
+    paths:
+      - '**.java'
+      - '**.xml'
+      - '**.yml'
+      - '**.yml'
```

```suggestion
      - '**.yml'
```

Please update the duplicate places, and it is recommended to splash before pushing